builderPodTemplate.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Pod
+spec:
+  imagePullSecrets:
+    - name: talend-registry
+  containers:
+    - name: talend-tsbi-dev
+      image: artifactory.datapwn.com/tlnd-docker-prod/talend/common/tsbi/jdk8-dev-base:2.9.4-20211124192604
```

Comment:
why jdk8 ?

Reply:
actually i don"t care about java version, just want the dev-base image